xla/xla.proto

```diff
@@ -711,15 +711,15 @@ message DebugOptions {
   // are counted.
   int64 xla_gpu_skip_mlir_kernels = 282;
 
-  // Let GEMM fusion autotuning probe cuDNN as a backend.
-  bool xla_gpu_cudnn_gemm_fusion = 277;
-
   // Threshold to rewrite matmul to cuBLAS or Triton (minumum combined number of
   // elements of both matrices in non-batch dimensions to be considered for a
   // rewrite).
   int64 xla_gpu_gemm_rewrite_size_threshold = 283;
 
-  // Next id: 284
+  // Let GEMM fusion autotuning probe cuDNN as a backend.
```

Comment:
Can you document the semantics of the integer here as well please? Let's also try to keep it more or less up-to-date in the future.

Reply:
Done. OK.